CURSOR.md

```
@@ -0,0 +1,235 @@
+```bash
+make ps
+make logs
+make new-logs
+make sh
+make down
+make stop
+```
+
+### Environment configuration
+This project relies on `.env` files (see `Makefile` includes). At minimum for local usage:
+```env
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)